openmetadata-ui/src/main/resources/ui/src/components/UserDataCard/UserDataCard.tsx

```diff
@@ -12,174 +12,64 @@
-        { 'tw-py-5': isDataset }
-      )}
+      className="tw-card tw-flex tw-gap-1 tw-py-2 tw-px-3 tw-group"
       data-testid="user-card-container">
-      <div className={`tw-flex ${isCheckBoxes ? 'tw-mr-2' : 'tw-gap-1'}`}>
-        {isIconVisible && !isDataset ? (
-          <Avatar name={item.description} />
-        ) : (
-          <>{getDatasetIcon(item.name)}</>
-        )}
-
-        <div
-          className={classNames('tw-flex tw-flex-col', {
-            'tw-pl-2': !isDataset,
-          })}
-          data-testid="data-container">
-          {isDataset ? (
-            <>{getDatasetTitle(item.name, item.description)}</>
-          ) : (
-            <>
-              <p
-                className={classNames('tw-font-normal', {
-                  'tw-cursor-pointer': Boolean(onClick),
-                })}
-                onClick={() => {
-                  onClick?.(item.id as string);
-                }}>
-                {item.description}
-              </p>
-              <p>{isIconVisible ? item.name : capitalize(item.name)}</p>
-            </>
-          )}
+      {item.profilePhoto ? (
+        <div className="tw-h-9 tw-w-9">
+          <img
+            alt="profile"
+            className="tw-rounded-full tw-w-full"
+            src={item.profilePhoto}
+          />
         </div>
-      </div>
-      {isActionVisible && (
-        <div className="tw-flex-none">
-          {isCheckBoxes ? (
-            <input
-              className="tw-px-2 custom-checkbox"
-              data-testid="checkboxAddUser"
-              type="checkbox"
-              onChange={() => {
-                onSelect?.(item.id as string);
-              }}
-            />
-          ) : (
-            <span
-              data-testid="remove"
-              onClick={() => onRemove?.(item.id as string)}>
-              <SVGIcons
-                alt="delete"
-                className="tw-text-gray-500 tw-cursor-pointer tw-opacity-0 hover:tw-text-gray-700 group-hover:tw-opacity-100"
-                icon="icon-delete"
-                title="Remove"
-                width="12px"
-              />
+      ) : (
+        <Avatar name={item.description} />
+      )}
+
+      <div
+        className="tw-flex tw-flex-col tw-flex-1 tw-pl-2"
+        data-testid="data-container">
+        <div className="tw-flex tw-justify-between">
+          <p
+            className={classNames('tw-font-normal', {
+              'tw-cursor-pointer': Boolean(onClick),
+            })}
+            onClick={() => {
+              onClick(item.id as string);
+            }}>
+            {item.description}
+          </p>
+          {!item.isActiveUser && (
+            <span className="tw-text-xs tw-bg-badge tw-border tw-px-2 tw-py-0.5 tw-rounded">
+              Inactive
             </span>
           )}
         </div>
-      )}
+        {/* <p>{isIconVisible ? item.name : capitalize(item.name)}</p> */}
```

Comment:
If this is not required then we should remove commented code.

Reply:
Updated.